### PR TITLE
Break out docs

### PR DESCRIPTION
**Pull request by patrickdent**
Separate docs form README

### CODE REVIEW

File: DOMAIN_MODEL.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Object MOdel
```
Comment (cherimarie) [inline DOMAIN_MODEL.md]: Typo "MOdel" -> "model"

File: DOMAIN_MODEL.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+## Character
+  * Name
+  * Contact Info
+  * Interests
+
+## Relationship
+  * Members (friends/characters)
+  * Score -2 - +2 (bed - good)
```
Comment (cherimarie) [inline DOMAIN_MODEL.md]: Typo: "bed" -> "bad"

Adding romantic flavor to relationships is v2 😆

File: REQUIREMENTS.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  * editable list of all recommended attendees
+  * form to add details about quest, date location etc
+  * convert outcome to quest, submission leads to view quest page
+* view quest page (quest)
+  * TODO list for invites
+  * details about party plan
+  * edit button
+  * delete button
+* View page for all Quests, filterable by past and upcoming
+* Characters, Relations, Parties and Quests are all permanently deletable.
```
Comment (cherimarie) [inline REQUIREMENTS.md]: "Relations" -> "Relationships"

File: DOMAIN_MODEL.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  * Name (optional)
+
+## Party
+  * Members
```
Comment (cherimarie) [inline DOMAIN_MODEL.md:13]: "Characters" rather than members, for consistency?

File: REQUIREMENTS.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+* Form to view characters.
+* Form to edit characters.
+* View page to view all characters.
+* View page to view all relationships, filterable by positive and negative.
+* Build a party form
+  * target number
+  * interests to include, ranked
+  * max/min of positive/negative relationships or total score
+  * sensible defaults for party, all counts in form are displayed as a range based on number possible
+  * submission of form leads to PArty Suggestino PAge
```
Comment (cherimarie) [inline REQUIREMENTS.md]: ToO maNy caPitalS :(

File: DOMAIN_MODEL.md (hunk 1 above)
Comment (patrickdent) [reply DOMAIN_MODEL.md]:
```suggestion
# Object Model
```

File: DOMAIN_MODEL.md (hunk 2 above)
Comment (patrickdent) [reply DOMAIN_MODEL.md]:
```suggestion
  * Score -2 - +2 (bad - good)
```

File: REQUIREMENTS.md (hunk 3 above)
Comment (patrickdent) [reply REQUIREMENTS.md]:
```suggestion
* Characters, Relationships, Parties and Quests are all permanently deletable.
```

File: DOMAIN_MODEL.md (hunk 4 above)
Comment (patrickdent) [reply DOMAIN_MODEL.md:13]:
```suggestion
  * Characters
```

File: REQUIREMENTS.md (hunk 5 above)
Comment (patrickdent) [inline REQUIREMENTS.md]:
```suggestion
  * submission of form leads to Party Suggestion Page
```

**Comment by cherimarie** [comment]
LGTM!

**Pull request merged**